clientv3/balancer.go

```diff
@@ -131,6 +147,59 @@ func (b *simpleBalancer) updateAddrs(eps []string) {
 	}
 }
 
+func (b *simpleBalancer) hasPinAddr(addrs []grpc.Address) bool {
+	for _, addr := range addrs {
+		if addr.Addr == b.pinAddr {
+			return true
+		}
+	}
+	return false
+}
+
+func (b *simpleBalancer) sendLoop() {
+	defer close(b.stopc)
```

Comment:
this isn't the convention; `stopc` is closed to signal to shutdown goroutines and `donec` is closed to signal that all goroutines are stopped